src/components/LikeButton/LikeButton.driver.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { StylableUnidriverUtil, UniDriver } from 'wix-ui-test-utils/unidriver';
+import style from './LikeButton.st.css';
+
+export interface LikeButtonDriver extends BaseUniDriver {
+  isMobile(): Promise<boolean>;
```

Comment:
isMobile can be removed. 
We might be able to just export the IconToggle's driver here, right?